src/site/content/en/progressive-web-apps/customize-install/index.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title: Provide a custom install experience
+authors:
+  - petelepage
+date: 2020-02-14
+updated: 2020-02-19
+description: |
+  Use the beforeinstallprompt event to provide a custom, seamless install
+  experience for your users.
+tags:
+  - post
+draft: true
+---
+
+Many browsers make it possible for you to enable and promote the installation
+of your Progressive Web App (PWA) directly within the user interface of your
+PWA. Installation (sometimes formerly referred to as Add to Home Screen),
+makes it easy for users to install your PWA on their mobile or desktop device.
+Installing a PWA adds it to a user's launcher, allowing it to be run like any
+other installed app.
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="spotify-custom-install.png"
+       alt="Install App button provided in the Spotify PWA">
+  <figcaption class="w-figcaption">
+    "Install App" button provided in the Spotify PWA
+  </figcaption>
+</figure>
+
+Why would you want a user to install your PWA? The same reason you'd want a
+user to install your app from any app store. Users who install are your most
+engaged users. Users who install a PWA have better engagement metrics than
+casual visitors, including more repeat visits, longer time on site and
+higher conversion rates, often at parity with native app users on mobile
+devices.
+
+<div class="w-clearfix"></div>
+
+## Promoting installation
+
+To indicate your Progressive Web App is installable, and to provide a custom
+in-app install flow:
+
+1. Listen for the `beforeinstallprompt` event.
+2. Save the `beforeinstallprompt` event, so it can be used to trigger the
+   install flow later.
+3. Alert the user your PWA is installable, and provide a button or other
+   element to start the in-app installation flow.
+
+### Listen for the beforeinstallprompt event
+
+If your Progressive Web App meets the required [installation criteria](/install-criteria/),
+the browser fires a `beforeinstallprompt` event. Save a reference to the
+event, and update your user interface to indicate that the user can install
+your PWA. This is highlighted below.
```

Comment:
```suggestion
your PWA.
```
I think readers have enough context to figure this out without an explicit callout, I'm also pretty sure that a11y best practice is to not refer to things as "above" or "below"